src/System.Net.Security/tests/UnitTests/System.Net.Security.Unit.Tests.csproj

```diff
@@ -57,4 +57,13 @@
     </Compile>
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <Target Name="RemoveProductionReference" BeforeTargets="CoreCompile">
+    <Message Text="Removing System.Net.Security reference before compiling" Importance="High" />
```

Comment:
You can probably remove the Message here, when we wrote this target this was only meant to prove that we were actually running the target before csc.exe :smile: